docs/adding-resources.md

```diff
@@ -34,10 +34,23 @@ configuration for the resource.
        make generate
        ```
 
-   6. Run against a Kubernetes cluster:
+   6. Register CRDs to cluster:
 
        ```console
        kubectl apply -f package/crds
+       ```
+   7. Run the provider against a Kubernetes cluster
```

Comment:
Thank you @sergenyalcin for figuring this out and correcting the documentation! Very much appreciated.

Reply:
Sure thing @ulucinar